### DISCUSSION TITLE
How to set custom color for background in wpf?

### DISCUSSION THREAD

**Discussion by VahidEra** [dotnet/wpf#7692] [Q&A] [answered]
Hi guys,
I have a button in a wpf project that I want to change its background color at runtime.
The only code that works is this:
```
btnMergNextWithDash.Background = System.Windows.Media.Brushes.BurlyWood
```
But the problem with this code is that I can't set a specific color in it. I tried different ways, but I still get the error Like this:

```
btnMergNextWithDash.Background = New SolidColorBrush(System.Windows.Media.Color.FromRgb(60, 179, 113))
```
Or:
```
btnMergNextWithDash.Background = New SolidColorBrush(System.Windows.Media.Color.FromRgb(60, 179, 113, 0))
```
But I still get the errors like "Too many arguments to Public overloads Sub New()"

Where is my problem and how can I do it?

**Comment by miloush** [comment, marked as answer]
I don't see anything wrong with what you do, but also there is no Sub, can you post a bit more of your code?

Edit: The last line is wrong though, FromRgb only takes 3 arguments and you have 4.

**Reply by VahidEra** [reply]
@miloush Dear Jan, In the last line it means:
```
btnMergNextWithDash.Background = New SolidColorBrush(System.Windows.Media.Color.FromArgb(60, 179, 113, 0))
```
But I solved the problem with your guidance.
After you said that there is no problem with the code, I thought of testing this code on another page, and it worked.
So I realized that The problem is only from the page where I enter the code, then with import ``` Imports SolidColorBrush = System.Windows.Media.SolidColorBrush``` The problem was solved.
As always, your advice helped me solve another problem. Thanks again for your advice, my dear friend.

**Reply by miloush** [reply]
Good to hear you solved your issue. Note that `FromArgb` has the alpha value first, and opaque is the maximum value, so `FromRgb(60, 179, 113) = FromArgb(255, 60, 179, 113)`.

**Reply by VahidEra** [reply]
@miloush Dear Jan, yes, I understood. Thanks again for your help.
